src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -444,7 +444,6 @@ def __call__(
         return_dict: bool = True,
         callback: Optional[Callable[[int, int, torch.FloatTensor], None]] = None,
         callback_steps: Optional[int] = 1,
-        **kwargs,
```

Comment:
I think we forgot to remove them when removing a deprecated function arg. They are never used anywhere